verify.rb

```diff
@@ -50,6 +50,12 @@ def test_img(img, name, imgs)
 end
 # rubocop:enable AbcSize,CyclomaticComplexity
 
+def output_ordered(websites)
+  websites.each do |site|
+    puts "  #{site['name']}\n"
+  end
+end
+ 
```

Comment:
still got a wee bit o' trailing whitespace

https://travis-ci.org/2factorauth/twofactorauth/jobs/353914396#L996

Reply:
made the change